Dockerfile

```diff
@@ -2,18 +2,18 @@ FROM python:3.9.12-buster as base
 WORKDIR /app
 RUN pip install poetry
 COPY pyproject.toml poetry.lock /app/
-RUN poetry install
+RUN poetry config virtualenvs.create false --local && poetry install
 COPY . /app/
 
-FROM base as production
-EXPOSE 8000
-RUN chmod +x /app/gunicorn.sh
-ENTRYPOINT [ "sh", "/app/gunicorn.sh" ]
-
 FROM base as development
 EXPOSE 5000
 RUN chmod +x /app/flask.sh
 ENTRYPOINT [ "sh", "/app/flask.sh" ]
 
 FROM base as test
-ENTRYPOINT [ "sh", "/app/test.sh" ]
\ No newline at end of file
+CMD [ "sh", "/app/test.sh" ]
+
+FROM base as production
+EXPOSE 8000
```

Comment:
Although you have this EXPOSE (which is good for documentation!), we're not actually using it so importantly your change to use `$PORT` instead of `8000` means the production image no longer works following your Readme commands, because the container doesn't know to use PORT 8000 by default
Adding a line here with the below would set that default so that users don't _need_ to explicitly pass through the `PORT` variable, but they can if they want to (e.g. Heroku)
```
ENV PORT=8000
```